src/server.h

```diff
@@ -151,6 +151,7 @@ struct hdr_histogram;
 #else
 #define CONFIG_ACTIVE_DEFRAG_DEFAULT 1
 #endif
+#define CLUSTER_NAMELEN 40
```

Comment:
If we need this, it's a sign of some bad design... I hope we can avoid it. This file doesn't include cluster.h and I think there is a reason to keep things separated.